src/cmd/sign.js

```diff
@@ -197,6 +200,7 @@ export default function sign(
           ...signSubmitArgs,
           apiUrlPrefix,
           apiProxy,
+          disableProgressBar: quiet,
```

Comment:
We'd need a test case to make sure that the value of the CLI flag is indeed passed to `signAddon()`.